scss/civicrm/api/_api.scss

```diff
@@ -0,0 +1,145 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('api')} {
+  .crm-container {
```

Comment:
@mukeshcompucorp  what about including this selector in `civi-page`? I thought we were going to do this since this selector appears every time we use this function?